draft-ietf-quic-recovery.md

```diff
@@ -350,17 +350,21 @@ endpoint:
   min_rtt.  This limits the underestimation that a misreporting peer can cause
   to the smoothed_rtt.
 
-On the first RTT sample for a network path, the smoothed_rtt is set to the
-latest_rtt.
-
-smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.  On
-the first RTT sample for a network path:
+smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.  When
+there are no samples for a network path:
```

Comment:
```suggestion
smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.

When there are no samples for a network path, and on the first RTT sample for
the network path:
```

Reply:
Splitting the text when having no samples and having one is intentional, see the changes suggested in https://github.com/quicwg/base-drafts/pull/3525#pullrequestreview-374825664.

I'm personally fine with the changes proposed here, but I would like to see what @ianswett thinks. Maybe it's OK, as the we have two paragraphs right after, discussing about each of the two conditions.

Reply:
I think @janaiyengar suggestion seems good and slightly clearer than before, though maybe line 361("where rtt_sample is the initial RTT." needs to be updated too?